src/common/rbac/const.go

```diff
@@ -62,19 +62,20 @@ const (
 	ResourcePreatInstance         = Resource("preheat-instance")
 	ResourceSelf                  = Resource("") // subresource for self
 
-	ResourceAuditLog           = Resource("audit-log")
-	ResourceCatalog            = Resource("catalog")
-	ResourceProject            = Resource("project")
-	ResourceUser               = Resource("user")
-	ResourceUserGroup          = Resource("user-group")
-	ResourceRegistry           = Resource("registry")
-	ResourceReplication        = Resource("replication")
-	ResourceDistribution       = Resource("distribution")
-	ResourceGarbageCollection  = Resource("garbage-collection")
-	ResourceReplicationAdapter = Resource("replication-adapter")
-	ResourceReplicationPolicy  = Resource("replication-policy")
-	ResourceScanAll            = Resource("scan-all")
-	ResourceSystemVolumes      = Resource("system-volumes")
-	ResourcePurgeAuditLog      = Resource("purge-audit")
-	ResourceExportCVE          = Resource("export-cve")
+	ResourceAuditLog            = Resource("audit-log")
+	ResourceCatalog             = Resource("catalog")
+	ResourceProject             = Resource("project")
+	ResourceUser                = Resource("user")
+	ResourceUserGroup           = Resource("user-group")
+	ResourceRegistry            = Resource("registry")
+	ResourceReplication         = Resource("replication")
+	ResourceDistribution        = Resource("distribution")
+	ResourceGarbageCollection   = Resource("garbage-collection")
+	ResourceReplicationAdapter  = Resource("replication-adapter")
+	ResourceReplicationPolicy   = Resource("replication-policy")
+	ResourceScanAll             = Resource("scan-all")
+	ResourceSystemVolumes       = Resource("system-volumes")
+	ResourcePurgeAuditLog       = Resource("purge-audit")
+	ResourceExportCVE           = Resource("export-cve")
+	ResourceJobServiceDashBoard = Resource("jobservice-dashboard")
```

Comment:
jobservice or jobservice-dashboard? stop an  jobservice-dashboard is a little wired.